helm_deploy/templates/deployment-clamav.yaml

```diff
@@ -0,0 +1,71 @@
+{{- /*
+  TODO: delete the below line and replace it with the commented out one
+  {{ if not .Values.branch }}
+*/ -}}
+{{ if .Values.branch }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: clamav
+spec:
+  replicas: {{ .Values.replicaCount.clamav }}
+  revisionHistoryLimit: 2
+  minReadySeconds: 15
+  serviceName: clamav
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 50%
+  selector:
+    matchLabels:
+      app: submit-crime-forms-clamav
+  template:
+    metadata:
+      labels:
+        app: submit-crime-forms-clamav
+        tier: backend
+    spec:
```

Comment:
We'll almost certainly want `allowPrivilegeEscalation: false` and `capabilities: drop: ["ALL"]`, otherwise if this deploys and runs fine it should be good

Reply:
@elken I originally tried having those here, and got `ValidationError(StatefulSet.spec.template.spec.securityContext): unknown field "allowPrivilegeEscalation"` and `ValidationError(StatefulSet.spec.template.spec.securityContext): unknown field "capabilities"`, so I moved those things to line 38 (the security context of the container as opposed to the template) and they seemed to work fine.

Reply:
Oh yes I see, okay I'm happy then. 

Do we want Joel or someone from cloud platform to look this over?

Reply:
@patrick-laa and @elken we could but I think if it deploys then we are unlikely to get any more from CP.